zdm/iteration.py

```diff
@@ -1623,12 +1626,32 @@ def minus_poisson_ps(log10C,data):
     return -lp
     
 
-def minimise_const_only(vparams:dict,grids,surveys,
-                        Verbose=True):
-    '''
+def minimise_const_only(vparams:dict,grids:list,surveys:list,
+                        Verbose=True, use_prev_grid:bool=True):
+    """
     Only minimises for the constant, but returns the full likelihood
     It treats the rest as constants
     the grids must be initialised at the currect values for pset already
+
+    Args:
+        vparams (dict): Parameter dict
+        grids (list): List of grids
+        surveys (list): List of surveys
```

Comment:
I think passing the surveys might be historical - originally information was different between the two, maybe we remove passing surveys if no info is needed anymore?

Reply:
Since the `survey` is held by the `grid`, it is superfluous.
But am keeping it around for now.